logic/src/commonMain/kotlin/com/wire/kalium/logic/data/client/ClientModel.kt

```diff
@@ -41,23 +41,15 @@ data class DeleteClientParam(
 
 data class Client(
     val id: ClientId,
-    val type: ClientType,
-    val registrationTime: String, // yyyy-mm-ddThh:MM:ss.qqq
-    val location: Location?,
+    val type: ClientType?,
+    val registrationTime: Instant?, // yyyy-mm-ddThh:MM:ss.qqq
     val deviceType: DeviceType?,
     val label: String?,
-    val cookie: String?,
```

Comment:
Those values are not used, please let me know if any are needed, especially for MLS @mchenani

Reply:
no, we don't need it as well!